tests/foreman/api/test_contentview.py

```diff
@@ -57,7 +57,7 @@ def test_subscribe_system_to_cv(self):
 
         # Promote the content view version.
         self.assertEqual(len(content_view.version), 1)
-        content_view.version[0].promote({u'environment_id': lc_env.id})
+        content_view.version[0].promote(data={u'environment_id': lc_env.id})
```

Comment:
Can a helper function be created in robottelo for the content view promote so I dont need to worry about carrying over this constant `environment_id` all over robottelo?

This will simplify this code to something like:

robottelo/robottelo/common/helpers.py
```py
def contentview_promote(contentview, env_id):
...
...
```

tests/foreman/api/test_contentview.py:
```py
contentview_promote(content_view.version[0], lc_env.id)
```